docs/src/componentDocs/Drawer/playground/PreviewComponent.tsx

```diff
@@ -185,95 +218,133 @@ export const PreviewComponent = (): JSX.Element => {
+                    >
+                        {variantIsTemporary && (
+                            <DrawerHeader
+                                title="Title"
+                                icon={<Close />}
+                                onClick={(): void => updateOpenProp(!drawerProps.open)}
+                                sx={{ cursor: 'pointer' }}
+                            />
+                        )}
+
+                        {variantIsPersistent && (
+                            <DrawerHeader
+                                title="Title"
+                                icon={<Menu />}
+                                onClick={(): void => updateOpenProp(!drawerProps.open)}
+                                sx={{ cursor: 'pointer' }}
+                            />
+                        )}
+
+                        {variantIsPermanent && <DrawerHeader title="Title" />}
+
+                        <DrawerBody>
+                            <DrawerNavGroup items={navGroupItems}></DrawerNavGroup>
+                        </DrawerBody>
+                    </Drawer>
+                    <Box
+                        sx={{
+                            backgroundColor: 'background.paper',
+                            height: 350,
+                            width: `calc(100vw - 850px)`,
+                            maxWidth: 700,
+                        }}
+                    >
+                        {variantIsRail ? (
+                            <Box sx={{ p: 2, ml: 9 }}>App Content Here.</Box>
+                        ) : (
+                            <>
+                                <AppBar position="static">
+                                    <Toolbar>
+                                        <IconButton
+                                            size="large"
+                                            edge="start"
+                                            color="inherit"
+                                            aria-label="menu"
+                                            sx={{ mr: 2 }}
+                                            onClick={(): void => updateOpenProp(!drawerProps.open)}
+                                        >
+                                            <Menu />
+                                        </IconButton>
+                                        <Typography variant="h6">Toolbar</Typography>
+                                    </Toolbar>
+                                </AppBar>
+                                <Box
+                                    sx={{
+                                        p: 2,
+                                        ml: drawerProps.open
+                                            ? `${drawerProps.width}px`
+                                            : drawerProps.variant === 'persistent'
+                                            ? 7
+                                            : 1,
```

Comment:
Temporary drawers should be "content-cover" instead of "content-push" — that is, when a temporary drawer is open, it should hover above the content, not push to the side.